input: change input repeat config

Kwiboo opened the pull request:
This PR changes the repeat delay and period for all LIRC remotes to match what Kodi sets for keyboards.

Currently `98-eventlircd.rules` sets the delay to 1000ms and the kernel sets the period to 33ms.
This is a too long delay and a too short period in my opinion.

~Copy the file to `/storage/.config/udev.rules.d/99-lircd-uinput.rules` and modify it to try with different delay/periods (after a reboot)~

HiassofT commented:
In my local setup I've removed the ir-keytable call from eventlircd completely - so the kernel default (500ms delay, 125ms repeat) is used which is quite fine IMO.

I'm not sure why the ir-keytable call is in 98-eventlircd.rule, IMO it should be removed from there and if LE decides that it needs a non-standard repeat/delay setting a rule should be added to the v4l-utils package (where ir-keytable and 70-infraread.rules) come from.

Kwiboo commented:
Updated to change the default delay set in `98-eventlircd.rules` to 500ms and only apply to input devices handled by eventlircd (excluding lircd-uinput)

Next update should move the delay/period option for lircd-uinput to `lirc_options.conf`

Kwiboo commented:
Updated to use `70-input-repeat.rules` to configure input repeat in a single file, thanks @HiassofT! (I changed to 80ms repeat period as the jump from current 33ms to 125ms was too noticeable).

This now also configures Kodi's advanced setting `remotedelay` to prevent Kodi from skipping repeat events. Made possible with upstream xbmc/xbmc#11663.

wrxtasy commented:
Feedback:
Both these commits have made a noticeable difference to the previously annoying mini delay on first IR remote control button pressing on the C2 with the original HardKernel remote and my custom Lirc Sony IR remote.

Thx @Kwiboo 👍

lrusak commented:
@Kwiboo did you want to change this back to 1000ms too?
